lib/node_modules/@stdlib/math/base/special/gammasgn/test/test.js

```diff
@@ -0,0 +1,165 @@
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of `gamma` for small positive numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = smallPositive.x;
+	expected = smallPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of `gamma` for medium positive numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = mediumPositive.x;
+	expected = mediumPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of `gamma` for small negative numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = smallNegative.x;
+	expected = smallNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of `gamma` for medium negative numbers', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var v;
+	var i;
+
+	x = mediumNegative.x;
+	expected = mediumNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		delta = abs( v - expected[ i ] );
+		tol = EPS * abs( expected[ i ] );
+		t.ok( delta <= tol, 'within tolerance. x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '. Delta: ' + delta + '. Tolerance: ' + tol + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var v = gammasgn( NaN );
+	t.equal( isnan( v ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `+0`', function test( t ) {
+	var v = gammasgn( +0.0 );
+	t.equal( isPositiveZero( v ), true, 'returns 0' );
+	t.end();
+});
+
+tape( 'the function returns `-0` if provided `+0`', function test( t ) {
+	var v = gammasgn( -0.0 );
+	t.equal( isPositiveZero( v ), true, 'returns -0' );
+	t.end();
+});
```

Comment:
We should add tests for negative integers. (i.e., we should return `0.0`). No need to use additional fixtures. Just create a `for` loop which tests against the first `100` negative integers.